pghoard/rohmu/object_storage/google.py

```diff
@@ -133,7 +133,8 @@ def _retry_on_reset(self, action):
                 elif isinstance(ex, HttpError):
                     # https://cloud.google.com/storage/docs/json_api/v1/status-codes
                     # https://cloud.google.com/storage/docs/exponential-backoff
-                    if ex.resp["status"] not in ("429", "500", "502", "503"):  # pylint: disable=no-member
+                    # 410: "Backend Error"
```

Comment:
https://cloud.google.com/storage/docs/json_api/v1/status-codes : 410—Gone
"""You have attempted to use a resumable upload session that is no longer available. If the reported status code was not successful and you still wish to upload the file, you must start a new session."""

I don't think it makes to retry uploads in this case.